integration-tests/src/dca.rs

```diff
@@ -1542,7 +1542,8 @@ mod stableswap {
 	#[test]
 	fn sell_should_work_with_stable_trades_and_omnipool() {
 		let amount_to_sell = 100 * UNITS;
-		let amount_to_receive = 70868187814642;
+		let amount_to_receive_1 = 70868187814642;
+		let amount_to_receive = 70832735995328;
```

Comment:
Never used. Remove the suffix `_1`.

Reply:
it is used. the _1 in the first run and the other variable in the second. 

now the amounts are different due to fee change.

Reply:
Ahh. Now I see.